mmeds/util.py

```diff
@@ -1583,3 +1600,86 @@ def get_subject_type(subject_file):
         if not sub_type == subject_type:
             subject_type = 'mixed'
     return subject_type.lower()
+
+
+def get_file_index_entry_location(path, tool, entry, testing=False):
+    """ Get entry from a non-active Tool's file_index.tsv file, for use in cross-tool analysis """
+
+    # Collect possible analysis dirs for given tool
+    dirs = path.glob(f"{tool}_*")
+    highest = 0
+    for d in dirs:
```

Comment:
this looks like the type of problem I'd use recursion for, not that it's unresonable to do it iteratively. Wondering if you considered both alternatives

Reply:
hm, no I hadn't. I'm not sure how I would go about this recursively, did you have something specific in mind?

Reply:
well you're iterating over directories, which usually lends itself to a recursive solution. now I gather that they're all at the same folder level? since file_indexes can exist at multiple levels and I wondered if that could trip up the code.